src/tests/api/ApiTestSteps.ts

```diff
@@ -181,7 +171,6 @@ export async function getSessionById(sessionId: string, tableName: string): Prom
 		session = Object.fromEntries(
 			Object.entries(originalSession).map(([key, value]) => [key, value.N ?? value.S]),
 		) as unknown as ISessionItem;
-		console.log("getSessionById Response", session);
```

Comment:
I know @Z-Mohamoud and @MohamedHassan101 leave logs in places to figure out why tests failing so just highlighting incase they want this kept

Reply:
Yeah happy to leave essential logs but they are pretty noisy so if there's anything we _can_ remove I'd like to do that. Let me know your thoughts Zac and Mohamed

Reply:
It's not that useful so happy for it to be removed